flang/lib/Lower/OpenMP.cpp

```diff
@@ -369,6 +369,22 @@ static void genOMP(Fortran::lower::AbstractConverter &converter,
     }
     iv = bounds->name.thing.symbol;
   }
+
+  for (const auto &clause : wsLoopOpClauseList.v) {
+    if (const auto &scheduleClause =
+            std::get_if<Fortran::parser::OmpClause::Schedule>(&clause.u)) {
+      if (const auto &chunkExpr{
```

Comment:
NIT: Please avoid using f18 style `{}`, For lowering code we use MLIR coding style :)

Reply:
Fixed